datasets/brp_azure/inschrijvingen/v2.0.0.json

```diff
@@ -0,0 +1,74 @@
+{
+  "id": "inschrijvingen",
+  "type": "table",
+  "version": "2.0.0",
+  "auth": "BRP/R",
+  "reasonsNonPublic": [
+    "5.1 1d: Bevat persoonsgegevens"
+  ],
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "identifier": "identificatie",
+    "required": [
+      "schema",
+      "identificatie"
+    ],
+    "display": "identificatie",
+    "properties": {
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.3.0#/definitions/schema"
+      },
+      "identificatie": {
+        "type": "string",
+        "description": "identificatie, dit is of het Burgerservicenummer, Anummer of een zelfgemaakte identificatie door GOB"
+      },
+      "datumIngangBlokkeringPL": {
+        "type": "string",
+        "format": "date",
+        "description": "datum waarop de gemeente de PL naar een andere gemeente stuurt"
```

Comment:
Wat is PL? misschien goed om uit te schrijven

Reply:
Bij deze gedaan